lib/lte_link_control/Kconfig

```diff
@@ -19,6 +19,11 @@ config LTE_AUTO_INIT_AND_CONNECT
 		automatically initialize and connect the modem
 		before the application starts
 
+config LTE_SHELL
+	bool "Enable lte shell commands"
+	default y if SHELL
```

Comment:
`if SHELL` is redundant here, as this config already depends on SHELL config - simple `default y` should be enough.